src/client/entity-editor/relationship-editor/actions.ts

```diff
@@ -30,10 +28,7 @@ export const UNDO_LAST_SAVE = 'UNDO_LAST_SAVE';
 
 export type Action = {
 	type: string,
-	payload?: mixed,
-	metadata?: {
-		debounce?: string
-	}
+	payload?: unknown
```

Comment:
No `meta?` here compared to the others. Is this on purpose?

Reply:
Yeah, meta isn't used in any of these actions. The underlying problem here is that we're not doing action typing in the greatest way, but to keep the diff small I haven't improved that here. I really like the approach [here](https://redux.js.org/recipes/usage-with-typescript), where there's a separate interface for each action type, and then a union type for all actions for a single dispatcher/redux component.

Reply:
Yes, that looks really clean.